app/scripts/components/data-catalog/index.tsx

```diff
@@ -156,15 +156,13 @@ function DataCatalog() {
       <LayoutProps
         title='Data Catalog'
         description={
-          getString('dataCatalogBanner')?.other ||
```

Comment:
Wait, this line tripped me into thinking that all the strings are object with `one` and `other`. But it is actually not based on ghg configuration? https://github.com/NASA-IMPACT/veda-config-ghg/blob/develop/veda.config.js#L41

Reply:
That is correct. All strings are an object, but to simplify configuration, the user can provide just a string. If that happens, the veda module will the use the same for `one` and `other`.
See here:

https://github.com/NASA-IMPACT/veda-ui/blob/6105ce04cf964e5df2eb3c9a70d111fc84d323dd/parcel-resolver-veda/defaults.js#L10-L20

Reply:
Documented at: https://github.com/NASA-IMPACT/veda-ui/blob/main/docs/content/CONFIGURATION.md#strings

Reply:
ok then it might be a bigger change if we want the type to correctly reflect the current data structure. So getString type is 
```
const getString: (variable: string) => {
    one: string;
    other: string;
  } | string;
```

And then anywhere that uses getString will need to be changed into

```
const storiesStringToUse = (typeof  getString('stories') ===string)?  getString('stories'): getString('stories').one
```

(Separating numeric string from plain string will be ideal, but I don't really want this PR to handle that issue. )

Reply:
How does `storiesBanner` string get translated in Veda module? If it is a string with a value 'aaa', it becomes `{one: 'aaa', other: 'aaa'}`? 

Also, it might be beneficial to mention I started this whole type thing because my automatic typechecker was giving an error for: `getString('storeisBanner)?.other`
Because `?` should not be necessary.  But when I get rid of `?` The app breaks because `getString('stroiesBanner')` was not guaranteed. So I wanted our type to reflect the data usage correctly, so the typechecker works automatically. Return type definition of `getString` got me think that the data should be defined as `{one: , other: }` form. (But if my assumption in the first paragraph is right, I guess it could be just a string value.).

Based on my confusion, wouldn't it be a better use of type if we actually separate numeric strings from non-numeric strings? I just pushed the change that treats numeric strings differently from the non-numeric ones.